sdk/core/core-rest-pipeline/src/pipeline.ts

```diff
@@ -129,7 +118,7 @@ interface PolicyGraphNode {
  * Do not export this class from the package.
  * @internal
  */
-class HttpPipeline implements Pipeline {
+export class HttpPipeline implements Pipeline {
```

Comment:
Rather than risk accidentally exposing this class, I'd rather use `createEmptyPipeline` inside `createPipelineFromOptions` instead